frontend/packages/dev-console/src/components/add/AddCardItem.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { SimpleListItem, Title, Text } from '@patternfly/react-core';
+import { ResolvedExtension, AddAction } from '@console/dynamic-plugin-sdk';
+import { useTelemetry } from '@console/shared/src/hooks/useTelemetry';
+import { navigateTo, resolvedHref } from '../../utils/add-page-utils';
+import { useShowAddCardItemDetails } from '../../hooks/useShowAddCardItemDetails';
+import './AddCardItem.scss';
+
+type AddCardItemProps = {
+  action: ResolvedExtension<AddAction>;
+  namespace: string;
+};
+
+const AddCardItem: React.FC<AddCardItemProps> = ({
+  action: {
+    properties: { id, label, icon, href, description },
+  },
+  namespace,
+}) => {
+  const fireTelemetryEvent = useTelemetry();
+  const [showDetails] = useShowAddCardItemDetails();
+
+  const actionIcon = (): JSX.Element => {
+    if (typeof icon === 'string') {
+      return (
+        <img
+          className="odc-add-card-item__header__icon"
+          src={icon}
+          alt={label}
+          aria-hidden="true"
+        />
+      );
+    }
+    if (typeof icon !== 'string' && React.isValidElement(icon)) {
+      return (
+        <span className="odc-add-card-item__header__icon" aria-hidden="true">
+          {icon}
+        </span>
+      );
+    }
+    return null;
+  };
+
+  return (
+    <div className="odc-add-card-item">
+      <SimpleListItem
+        onClick={(e: React.SyntheticEvent) => {
+          fireTelemetryEvent('Add Item Selected', {
+            id,
+            name: label,
+          });
+          navigateTo(e, resolvedHref(href, namespace));
+        }}
+        href={resolvedHref(href, namespace)}
+        data-test-id="odc-add-card-item"
```

Comment:
As the "item" is the complete AddCardItem component I would recommend to call this `link`, so that it's clear which part is "clickable". 

```suggestion
        data-test-id="odc-add-card-link"
```

Reply:
done